### ISSUE TITLE
Improvements on tabbing interaction for the Forum

### ISSUE THREAD

**Issue by bmcutler** [Submitty/Submitty#5776]
1.  Currently, when tabbing through the thread list, we alternate between focus on a specific thread, and a hidden button "skip into thread".   By default, the top thread is selected/open, and the posts from that thread are shown on the right.  If we tab halfway down the thread list, and press "skip into thread", we will be on the posts from the already selected/open thread.  It would be more intuitive to have "skip into thread" both select/open the thread in the middle of the list (so its posts are displayed on the right side) so that then when we jump to the right side, we can navigate o those posts.

2. Reply box to post on a thread should have our standard "Press TAB to indent. Press ESC to advance from textarea." (e.g., appears on the notebook boxes).  Let's standardize this text & styling with a wrapper function around these input boxes that is used everywhere on the site.   Perhaps instead of just simple black text (which can get confused with instructor markdown), we can make it italics & dark grey? (design discussion)  @Kevin-MM  will help here -- and look for other places on the site missing this.

3. When your focus is on the bottom of the post list (in the reply box) and you shift tab (go backwards) it would be nice to have an invisible button "skip back to first post" rather than the current option to "skip back to thread list".
(design discussion?)

[ from discussion with @holzbh ]
